src/core/blueprint/into_schema.rs

```diff
@@ -107,9 +107,10 @@ fn to_type(def: &Definition) -> dynamic::Type {
                                 let ctx: ResolverContext = ctx.into();
                                 let ctx = EvalContext::new(req_ctx, &ctx);
 
-                                FieldFuture::from_value(
-                                    ctx.path_value(&[field_name]).map(|a| a.into_owned()),
-                                )
+                                match ctx.path_value(&[field_name]).map(|a| a.into_owned()) {
+                                    Some(ConstValue::Null) => FieldFuture::Value(FieldValue::NONE),
+                                    a => FieldFuture::from_value(a),
+                                }
```

Comment:
maybe we can not do this here, there might be some cases where we need to throw an error in the case of Null

Can you explain why this change makes more sense to you then the changes you made at async-graphql?

Reply:
I have a use case where an internal type ( which is enum is null ) with the graphql directive. the reoslution always fails with error. if you check the test case it will fails without such modification

Reply:
```
-old snapshot
+new results
────────────┬────────────────────────────────────────────────────────────────────
    2     2 │   "headers": {
    3     3 │     "content-type": "application/json"
    4     4 │   },
    5     5 │   "body": {
    6       │-    "data": {
    7       │-      "post": {
    8       │-        "severity": {
    9       │-          "type": null
   10       │-        }
          6 │+    "data": null,
          7 │+    "errors": [
          8 │+      {
          9 │+        "message": "internal: invalid item for enum \"EnumType\"",
         10 │+        "locations": [
         11 │+          {
         12 │+            "line": 1,
         13 │+            "column": 27
         14 │+          }
         15 │+        ],
         16 │+        "path": [
         17 │+          "post",
         18 │+          "severity",
         19 │+          "type"
         20 │+        ]
   11    21 │       }
   12       │-    }
         22 │+    ]
   13    23 │   }
   14    24 │ }
```

Reply:
@ssddOnTop I added another test to make sure that it will fail with exception when it is mandatory